matching-service/package.json

```diff
@@ -9,23 +9,25 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "npx eslint .",
     "lint-fix": "npx eslint . --fix",
-    "migrate": "node_modules/.bin/sequelize db:migrate"
+    "migrate": "./node_modules/.bin/sequelize db:migrate"

```

Comment:
The migrate function doesn't seem to be working on my side. It says '.' is not a recognized command. I was wondering if it is related to this. https://stackoverflow.com/questions/66185466/npm-script-fails-is-not-recognized-as-an-internal-or-external-command